frontend/components/transaction/TransactionMessage.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <TextBlock :title="isExplanationsShown ? 'Make a bid' : ''">
+        <div v-if="transactionAddress">
+            Transaction <format-address shorten :value="transactionAddress" /> was successfully executed.
+            <format-address explicit :value="transactionAddress" />
+        </div>
+        <div v-else-if="isExplanationsShown">
+            Auction bidding incurs transaction fee of approximately
+            <FormatCurrency :value="transactionFee" :decimals="5" currency="ETH" />, hence, the connected wallet needs
+            to hold enough funds to cover these fees. The transaction fee is a recommended value and based on the
+            <Explain text="average fast fee">
+                <a href="https://ethereum.org/en/developers/docs/gas/#why-can-gas-fees-get-so-high" target="_blank">
+                    Gas Prices can change
+                </a>
+                due to changing busyness of the Ethereum network. However, you can look up the current average gas
+                price
+                <a href="https://ethgasstation.info/" target="_blank"> here </a> </Explain
+            >. The amount can be edited by the participant to influence the speed of the transaction. <br /><br />
+            By default, the profit from a successful bidding will be sent to the same wallet which executed the
+            transaction. However, you can provide a different address which will receive the transaction profit.
```

Comment:
- [ ] This part of the message should not be shown for the new `BidBlock` as it does not apply to the Dai bidding flow but only to the swapping based flow that we have implemented in the past. (so please remove here)